test/bin/ipfs-pin-stat

```diff
@@ -1,30 +0,0 @@
-#!/bin/sh
```

Comment:
It was not a test. It was a script used only by t0081.
Now we can do the same thing with `ipfs pin ls <arg>...` so this script is not needed any more.